### PR TITLE
Feature/ondergrond dossier

### PR DESCRIPTION
**Pull request by thomasm0**

### CODE REVIEW

File: src/pages/static/UndergroundDossierPage/components/Intro/Intro.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  }
+`
+
+const StyledIntroContent = styled.div`
+  margin: 0 auto;
+  max-width: 920px;
+`
+
+const StyledIntroParagraph = styled(Paragraph)`
+  margin: 0 ${themeSpacing(4)};
```
Comment (lpsmitamsterdam) [inline src/pages/static/UndergroundDossierPage/components/Intro/Intro.tsx:30]: I think the paragraph shouldnt have horizontal margin to align with the h1 on desktop

Comment (lpsmitamsterdam) [reply src/pages/static/UndergroundDossierPage/components/Intro/Intro.tsx:30]: Schermafbeelding 2022-01-28 om 10 05 36

File: src/pages/static/UndergroundDossierPage/components/Intro/Intro.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { breakpoint, Heading, Paragraph, themeSpacing } from '@amsterdam/asc-ui'
+import type { FunctionComponent } from 'react'
+import styled from 'styled-components'
+
+const StyledIntro = styled.section`
+  padding: ${themeSpacing(6)} 0;
+  background-color: #fafafa;
```
Comment (lpsmitamsterdam) [inline src/pages/static/UndergroundDossierPage/components/Intro/Intro.tsx:7]: replace with asc variable?

**Review by lpsmitamsterdam — CHANGES_REQUESTED** [review]

File: src/pages/static/UndergroundDossierPage/components/Intro/Intro.tsx (hunk 2 above)
Comment (thomasm0) [reply src/pages/static/UndergroundDossierPage/components/Intro/Intro.tsx:7]: The closest ASC color is `#F5F5F5` which is a little darker than the design's `#FAFAFA`

**Review by lpsmitamsterdam — APPROVED** [review]

**Pull request merged**